src/fem/fe_values.hh

```diff
@@ -245,7 +219,7 @@ public:
      *
      * @param point_no Number of the quadrature point.
      */
-	inline arma::vec::fixed<spacedim> normal_vector(unsigned int point_no) override
+	inline arma::vec::fixed<spacedim> normal_vector(unsigned int point_no)
```

Comment:
To discuss: Normal has no sense for the bulk quadrature points.

Reply:
Should be resolved together with new mechanism of cached values and point sets.